src/aggregate/aggregate_request.c

```diff
@@ -329,6 +329,13 @@ static int handleCommonArgs(AREQ *req, ArgsCursor *ac, QueryError *status, int a
     if (parseValueFormat(&req->reqflags, ac, status) != REDISMODULE_OK) {
       return ARG_ERROR;
     }
+  } else if (AC_AdvanceIfMatch(ac, "_INDEX_PREFIXES")) {
+    // Set the offset of the prefixes in the query, for further processing later
+    req->prefixesOffset = ac->offset - 1;
+
+    // Advance by the number of prefixes + 1 (for the n_prefixes argument)
+    long advance_by = strtol((const char *)AC_CURRENT(ac), NULL, 10);
+    AC_AdvanceBy(ac, advance_by + 1);
```

Comment:
Since we are constructing the command, we know the args exist? (and no need to check return code)

Reply:
Yes